### PR TITLE
Alternative path for .node files

### PR DESCRIPTION
**Pull request by chagong**
Alternative path for search .node files.

### CODE REVIEW

File: lib/detector.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 const path = require("path");
-const detectionNodeDestPath = path.join(__dirname, `../../usb-detection/build/Release/detection.node`);
+let detectionNodeRelativePath = typeof __webpack_require__ === "function" ? `../node_modules/usb-detection/build/Release/detection.node` : `../../usb-detection/build/Release/detection.node`;
+const detectionNodeDestPath = path.join(__dirname, detectionNodeRelativePath);
```
Comment (dilin-MS) [inline lib/detector.js]: Can we put this into an utility function like `getPath()`(or another better name) which focus on getting path concerning the require condition (webpack require or not).

Comment (chagong) [reply lib/detector.js]: You mean add a utils.js for that, don't you?

Comment (dilin-MS) [reply lib/detector.js]: Yes.

Comment (chagong) [reply lib/detector.js]: Done.

File: lib/utils.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const path = require("path");
+
+function getNodeFilePath(relativePath) {
```
Comment (dilin-MS) [inline lib/utils.js]: You can add comment for the function to explain the usage.

**Comment by chagong** [comment]
Travis CI didn't catch the latest commit to run build. I will cancel this PR and re-open a new PR.

**Pull request closed without merging**